diff --git a/Content/Product/DataAbstractionLayer/VariantListingUpdater.php b/Content/Product/DataAbstractionLayer/VariantListingUpdater.php
index 0ee16dc..1dcb55d 100644
--- a/Content/Product/DataAbstractionLayer/VariantListingUpdater.php
+++ b/Content/Product/DataAbstractionLayer/VariantListingUpdater.php
@@ -90,6 +90,7 @@ class VariantListingUpdater
                 $query->innerJoin('root', 'product_option', $mappingAlias, $mappingAlias . '.product_id IS NOT NULL');
                 $query->innerJoin($mappingAlias, 'property_group_option', $optionAlias, $optionAlias . '.id = ' . $mappingAlias . '.property_group_option_id AND ' . $optionAlias . '.property_group_id = :' . $optionAlias);
                 $query->andWhere($mappingAlias . '.product_id = product.id');
+                $query->setMaxResults(1);

                 $fields[] = 'LOWER(HEX(' . $optionAlias . '.id))';
